src/main/java/ti4/map/Game.java

```diff
@@ -161,6 +161,12 @@ public class Game {
     private boolean miltyModMode;
     @Getter
     @Setter
+    private boolean promisesPromises;
+    @Getter
+    @Setter
+    private boolean flagshipping;
+    @Getter
+    @Setter
```

Comment:
GameSaveLoadManager.java will need to explicitly save these values, akin to other booleans